subsys/bluetooth/controller/ll_sw/nordic/lll/lll_test.c

```diff
@@ -698,6 +708,7 @@ uint8_t ll_test_rx(uint8_t chan, uint8_t phy, uint8_t mod_idx, uint8_t expected_
 #endif /* CONFIG_BT_CTLR_DTM_HCI_DF_IQ_REPORT */
 
 	radio_pkt_rx_set(radio_pkt_scratch_get());
+	radio_tmr_tifs_set(EVENT_IFS_US);
```

Comment:
Have this unrelated change in separate commit.

Reply:
It is related, because tifs was set previously in init function for both Tx and Rx tests. However now we start use a tifs for Tx test with different value so I needed to move it here for Rx test.